src/commands/InitCommand.ts

```diff
@@ -33,6 +34,10 @@ export class InitCommand implements yargs.CommandModule {
             })
             .option("docker", {
                 describe: "Set to true if docker-compose must be generated as well. False by default."
+            })
+            .option("pm", {
+                alias: "manager",
+                describe: "Install packages, expected values are npm or yarn."
```

Comment:
If these are the valid values, should we set `choices` to these two?

Reply:
done